examples/README.md

```diff
@@ -27,4 +27,58 @@ Run the example:
 
 ```
 ./examples/agent/run.sh
-```
\ No newline at end of file
+```
+
+## FireLens on ECS
+
+You can deploy the example by running the following:
+
+```sh
+# create an ECR repository for the example image
+aws ecr create-repository --repository-name <image-name> --region <region>
+
+# create an S3 bucket for the Fluent-Bit configuration
+aws s3api create-bucket --bucket <bucket-name> --region <region>
+
+# create ECS cluster
+# create ECS task definition
+# create ECS service
+
+# deploy
+./examples/ecs-firelens/publish.sh \
+  <account-id> \
+  <region> \
+  <image-name> \
+  <s3-bucket> \
+  <ecs-cluster-name> \
+  <ecs-task-family> \
+  <ecs-service-name>
+```
+
+### Example Metrics
+
+```json
+{
+  "_aws": {
+    "Timestamp": 1583902595342,
+    "CloudWatchMetrics": [
+      {
+        "Dimensions": [[ "ServiceName", "ServiceType" ]],
+        "Metrics": [{ "Name": "ProcessingTime", "Unit": "Milliseconds" }],
+        "Namespace": "aws-embedded-metrics"
+      }
+    ]
+  },
+  "ServiceName": "example",
```

Comment:
where does the service name come from?

Reply:
During environment detection, we'll fetch some metadata from the ECS Instance metadata. This is coming from the image name: https://github.com/awslabs/aws-embedded-metrics-node/pull/28/files#diff-4b54ad9e319d6fb8a8c809af1c06de47R88

Reply:
Hmmm, I don't love that because in most cases that means it will end up being an ECR image URI which is ugly, but I suppose this might be the best unique identifier. I wish Task Metadata gave you the ECS Service name. This is a bit confusing because the ServiceName is not the name ECS Service.

Reply:
So, I've added [some formatting](https://github.com/awslabs/aws-embedded-metrics-node/pull/28/commits/36a3f61c7ef16b6da3cfb704e9fc15a9f96650f8#diff-4b54ad9e319d6fb8a8c809af1c06de47R54) so we get the image and tag instead of the full URI (e.g. `image-name:latest`). I agree that I'd prefer to have the actual ECS service name, but I don't know of a way to get this. For additional context ServiceName and ServiceType are dimensions used by CloudWatch Service Lens to identify a unique service which is why these are set by default.